mod-clans/src/main/java/gg/solarmc/loader/clans/ClanMember.java

```diff
@@ -29,7 +29,7 @@
  *
  * @param userId the user ID of the member
  */
-public record ClanMember(int userId) {
+public record ClanMember(int userId) { //wait since when was this a record :thinking:
```

Comment:
Please, I don't want to see `:thinking:` anymore in code or writing

Reply:
why not

Reply:
it doesn't help me when reading code